include/circt/Conversion/FIRRTLToLLHD/Passes.td

```diff
@@ -22,6 +22,7 @@ def ConvertFIRRTLToLLHD : Pass<"convert-firrtl-to-llhd", "ModuleOp"> {
     description.
   }];
   let constructor = "circt::llhd::createConvertFIRRTLToLLHDPass()";
+  let dependentDialects = ["llhd::LLHDDialect"];
```

Comment:
Nit: should this and other modifications use the full package name like the line above this addition?

Reply:
Good question!  I'm not totally sure, but this is what MLIR consistently does.  It generates a call to the constructor in `Passes.h.inc` which is usually `#include`d inside of the `circt` namespace, so I think theoretically we don't need to use the `circt` namespace prefixed on either of these.

Unrelated, but this did just make me realize that the conversion passes are usually not in the dialect namespace:
```diff
-circt::llhd::createConvertFIRRTLToLLHDPass
+circt::createConvertFIRRTLToLLHDPass
```

Reply:
I'm fine with either way and no need to gate this PR. I'm good with it as is!